sky/cli.py

```diff
@@ -4134,7 +4135,9 @@ def storage_delete(names: List[str], all: bool, yes: bool, async_call: bool):  #
             return
         names = [storage['name'] for storage in storages]
     else:
-        names = _get_glob_storages(names)
+        storages = sdk.get(sdk.storage_ls())
```

Comment:
Maybe add a TODO that we can add an argument in `sdk.storage_ls` SDK to filter it on the server side for saving less info transmitted.